wpe-runtime/org.wpe.Sdk.json

```diff
@@ -150,13 +137,21 @@
                     "type": "archive",
                     "url": "https://wpewebkit.org/releases/wpewebkit-2.22.3.tar.xz",
                     "sha256": "241f177cded1a4ba3088716650eaaac25588878882170789b3c630ac45dbb2fa"
+                },
+                {
+                    "type": "patch",
+                    "path": "wpe-webkit/0001-Add-the-WebKitVersion.h-header-https-bugs.webkit.org.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "wpe-webkit/0002-WPE-Add-the-WebKitVersion.h-header.patch"
                 }
             ],
             "config-opts": [
                 "-DPORT=WPE",
                 "-DCMAKE_BUILD_TYPE=Release",
-                "-DCMAKE_C_FLAGS_RELEASE=-O2 -DNDEBUG",
```

Comment:
What flags is WebKit built with then? Are optimizations enabled elsewhere?

Reply:
Yes. This is the current flags:
```
FLAGS = -fdiagnostics-color=always -Wno-expansion-to-defined -Wno-noexcept-type -Wno-maybe-uninitialized -Wwrite-strings -Wundef -Wpointer-arith -Wmissing-format-attribute -Wformat-security -Wcast-align -Wextra -Wall -march=x86-64 -mtune=generic -O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fno-strict-aliasing -fno-exceptions -fno-rtti -std=c++14 -O3 -DNDEBUG -fPIC
```